src/Stratis.Bitcoin.Features.Consensus/CoinViews/CoinView.cs

```diff
@@ -14,9 +15,9 @@ public abstract class CoinView
         /// Retrieves the block hash of the current tip of the coinview.
         /// </summary>
         /// <returns>Block hash of the current tip of the coinview.</returns>
-        public async Task<uint256> GetBlockHashAsync()
+        public async Task<uint256> GetBlockHashAsync(CancellationToken ct = default(CancellationToken))
```

Comment:
would be nice if you added parameter comment if you are adding parameter to a commented method

Reply:
Honestly I don't see any relevant comment I can put here :(

Reply:
try `Cancellation token to allow the caller to cancel the execution of the operation.`
or `Cancellation token to observe while waiting for the task to complete.`

Reply:
Honestly I think this would just add noise, cancellationTokens are always passed in for that purpose, so unless we use it in an unconventional way, why explain ?

Reply:
This PR was supposed to be a change of one line. You changed 250 lines and now you don't like adding noise? This whole PR is noise. So at least if you add something, add it properly. Or just delete the whole PR, I don't care. We have https://github.com/stratisproject/StratisBitcoinFullNode/blob/master/Documentation/commenting-policy.md or if you like the new version - https://github.com/stratisproject/StratisBitcoinFullNode/pull/1678 - pick one you like. If you don't like it, open discussion elsewhere, don't discuss this on PR.